### DISCUSSION TITLE
data folder time not consitent

### DISCUSSION THREAD

**Discussion by freesinger** [prometheus/prometheus#11751] [Q&A] [answered]
### What did you do?

I am using systemd service to manage prometheus,
![image](https://user-images.githubusercontent.com/20896930/208837992-48b80992-8164-4479-ab14-ef35b7ec37d5.png)
but I found time of the folders in data directory is not consitent, is this because the underlying compaction operation of Prometheus?

### What did you see instead? Under which circumstances?

![image](https://user-images.githubusercontent.com/20896930/208838662-d672fdeb-46e0-4f24-be8e-1be35b8c7395.png)

### System information

Linux 5.10.84-10.4.al8.x86_64 x86_64

### Prometheus version

```text
prometheus, version 2.36.1 (branch: HEAD, revision: 3c0a100dec2950f23e91f7efbf8867750960807d)
  build user:       root@e59c79158868
  build date:       20220609-15:25:21
  go version:       go1.18.3
  platform:         linux/amd64
```

**Comment by roidelapluie** [comment, marked as answer]
Yes, it is normal for the timestamp of blocks in Prometheus to not be chronological. As you noted, this is because of the way compaction works in Prometheus. This should not be a cause for concern.

**Reply by freesinger** [reply]
Got it, thx
